### ISSUE TITLE
Do you actually use the Lion optimizer?

### ISSUE THREAD

**Issue by StrangeTcy** [mosaicml/llm-foundry#145]
Your [post ](https://www.mosaicml.com/blog/mpt-7b) says:

> How did we do this? First, we addressed convergence stability with architecture and optimization improvements. Our MPT models use [ALiBi](https://arxiv.org/abs/2108.12409) rather than positional embeddings, which we found to improve resilience to loss spikes. We also train our MPT models with [the Lion optimizer](https://arxiv.org/abs/2302.06675) rather than AdamW, which provides stable update magnitudes and cuts optimizer state memory in half.

Meanwhile, your [training config](https://github.com/mosaicml/llm-foundry/blob/main/scripts/train/yamls/pretrain/mpt-7b.yaml#L60) says:
`name: decoupled_adamw`.

Am I missing something here?

ETA: judging by [search results](https://github.com/search?q=repo%3Amosaicml%2Fllm-foundry%20decoupled_lionw&type=code), config processing is the only place where Lion is mentioned, that is to say, it's not used in any of the configs github can find.
